src/Calendar.js

```diff
@@ -180,6 +188,7 @@ class Calendar extends Component {
           isSelected={ isSelected || isEdge }
           isInRange={ isInRange }
           key={ index }
+          isPassive = { isPassive || isOutsideMinMax }
```

Comment:
`isPassive` is a property on the `data` parameter that is passed into the `map` function. 

`const { dayMoment, isPassive } = data`.